### DISCUSSION TITLE
Use generated certificate in another container

### DISCUSSION THREAD

**Discussion by dj-fiorex** [nginx-proxy/acme-companion#1029] [Q&A]
Hello to all, i have a question about using the certificates generated by acme-companion in another container (mosquitto) of a docker-composer.
Right now i have a shared volume and the mosquitto container uses this volume to be able to read the certificates but I thing it is an hacky way because i also need to set the permissions of the certificates to the userId of the mosquitto user. 
Is there a clean way to be able to use the same certificates with mosquitto or another container? 

Thanks!

**Comment by Tockra** [comment]
Hey,

I would be interest in a solution. My keycloak docker container uses internally the userid 1000. 
I mounted the certs directory to the keycloak container. But it has problems to start because of permission problems. (user 1000 has no read permissions to the certs). 
Further I can't change the owner of the certificates. It is changed back to root automatically.

**Reply by Tockra** [reply]
@dj-fiorex did you found a solution?

**Reply by dj-fiorex** [reply]
Unfortunately, no. I switched to traefik

**Reply by Tockra** [reply]
> Unfortunately, no. I switched to traefik

Okay thank you for the quick answere. I will give traefik a try too.
